ACM_General/payments/managers.py

```diff
@@ -8,6 +8,9 @@ class TransactionCategoryManager(models.Manager):
     """
     def get_by_natural_key(self, name):
         """
+        :param name: The name of Transaction.
+        :type name: string
```

Comment:
The :type param: should be the standard python types. For example `string=str` or `dictionary=dict`, etc. What I mean by 'standard python types' is the function you would use to transform a type into another type
```python
dict()
list()
str()
int()
```